Gemfile.base

```diff
@@ -91,10 +91,10 @@ gem 'analytics-ruby', require: false
 group :development, :test do
   gem 'bootsnap', '~> 1.3'
 
-  platform :mri_23, :mri_24, :mri_25 do
+  platform :mri_24, :mri_25 do
     gem 'pry-byebug', '>= 3.7.0', require: false
     gem 'pry-stack_explorer', require: false
-  end
+end
```

Comment:
Just align this `end` with the `platform`

Reply:
Fixed